tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -496,14 +496,44 @@ public function testEventRendering(): void {
     $this->assertSession()->linkNotExistsExact('Main link for media items');
     $this->assertSession()->pageTextNotContains('More media links');
 
-    // Assert "Full text", "Featured media", "Featured media legend" fields
-    // (these fields have to be filled all together).
+    // Assert "Description" title is not rendered unless there is a body text.
+    $this->assertSession()->pageTextNotContains('Description');
+
+    // Assert "Full text", "Featured media", "Featured media legend" fields..
     $node->set('body', 'Event full text');
+    $node->save();
+    $this->drupalGet($node->toUrl());
+
+    $description_content = $this->assertSession()->elementExists('css', 'article > div > div:nth-child(3)');
+    $text_featured = new TextFeaturedMediaAssert();
+    $text_featured_expected_values = [
+      'title' => 'Description',
+      'text' => 'Event full text',
+    ];
+    $text_featured->assertPattern($text_featured_expected_values, $description_content->getHtml());
+
+    $node->set('body', '');
```

Comment:
Please adapt the test coverage without the changes in oe_content because having a media without description will break the layout and that's fixed in EWPP-2714. This ticket should only fix the issue regarding having a heading for the description field group when no field is filled in.
So, only the first assertion is needed as a change to make sure that without having any of the fields we don't render a text with featured media pattern.